analytic_engine/src/sst/parquet/writer.rs

```diff
@@ -183,14 +187,23 @@ impl RecordBatchGroupWriter {
         )
         .box_err()
         .context(EncodeRecordBatch)?;
+        let mut parquet_filter = if self.level.is_min() {
```

Comment:
Shall we provide a method called `should_build_parquet_filter` which decided by `!self.level.is_min() && !self.hybrid_encoding`? After that, we can remove the option wrapper of the return value of `build_row_group_filter` method, which will look pretty.

Reply:
Good suggestion